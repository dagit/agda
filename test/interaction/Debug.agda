{-# OPTIONS -v5 #-}

module Debug where

postulate Foo : Set
