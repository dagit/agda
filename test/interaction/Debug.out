(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Debug (Debug.agda).\n" t)
(agda2-verbose "Building interface...\n")
(agda2-verbose "Writing interface file Debug.agdai.\n")
(agda2-verbose "Wrote interface file.\n")
(agda2-info-action "*Type-checking*" "Finished Debug.\n" t)
(agda2-verbose "  New module. Let's check it out.\n")
(agda2-verbose "  Now we've looked at Debug\n")
(agda2-status-action "Checked")
(agda2-info-action "*All Goals*" "" nil)
((last . 1) . (agda2-goals-action '()))
